# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../linux-header/xt_layer7.h.diff
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- linux-2.6.27.6/include/linux/netfilter/Kbuild.orig	2008-11-26 17:11:17.000000000 +0200
+++ linux-2.6.27.6/include/linux/netfilter/Kbuild	2008-11-26 19:13:14.000000000 +0200
@@ -23,6 +23,7 @@
 header-y += xt_hashlimit.h
 header-y += xt_iprange.h
 header-y += xt_helper.h
+header-y += xt_layer7.h
 header-y += xt_length.h
 header-y += xt_limit.h
 header-y += xt_mac.h
